i6_models/samplers/log_uniform.py

```diff
@@ -0,0 +1,34 @@
+__all__ = ["LogUniformSampler"]
+
+
+import torch
+from torch import nn
+from typing import Optional
+
+
+class LogUniformSampler(nn.Module):
```

Comment:
This class is a subclass of nn.Module, which makes it a part and thus should be in the parts folder instead of a dedicated samplers folder.